docs/reference/filebeat/filebeat-input-filestream.md

```diff
@@ -317,25 +317,50 @@ The setting relies on the modification time of the file to determine if a file i
 
 To remove the state of previously harvested files from the registry file, use the `clean_inactive` configuration option.
 
+## Take over [filebeat-input-filestream-take-over]
+When `take_over` is enabled, this `filestream` input will take over
+states from the [`log`](/reference/filebeat/filebeat-input-log.md) input
+or other `filestream` inputs. Only states of files being actively
+harvested by this input are taken over.
 
-#### `take_over` [filebeat-input-filestream-take-over]
+To take over states from the log input, simply set `take_over.enabled:
+true`. To take over states from other `filestream` inputs, set
+`take_over.enabled: true` and set `take_over.from_ids` as list of
+previous `filestream` IDs you want to migrate states from.
 
-If `take_over` is set to `true`, this `filestream` will take over all files from `log` inputs if they match at least one of the `paths` set in the `filestream`.
+```yaml
+take_over:
+  enabled: true
+  from_ids: ["foo", "bar"] # omit to take over from the log input
+```
+:::{important}
+Take over can only be used if the inputs it is taking states from are
+not active any more. If old inputs harvesting the file are still
+active, the data ingestion will be inconsistent (data duplication or
+data loss will occur).
```

Comment:
```suggestion
The `take over` mode can work correctly only if the source (taken from) inputs are no longer active. If source inputs are still harvesting the files which are being migrated, it will lead to data duplication and in some cases might cause data loss.
```